app/templates/admin/events.hbs

```diff
@@ -1 +1,27 @@
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="sixteen wide column">
+      {{#tabbed-navigation isNonPointing=true}}
+        {{#link-to 'admin.events.index' class='item'}}
+          {{t 'All Events'}}
+        {{/link-to}}
+        {{#link-to 'admin.events.list' 'live' class='item'}}
+          {{t 'All Live'}}
+        {{/link-to}}
+        {{#link-to 'admin.events.list' 'draft' class='item'}}
+          {{t 'All Draft'}}
+        {{/link-to}}
+        {{#link-to 'admin.events.list' 'past' class='item'}}
+          {{t 'All Past'}}
+        {{/link-to}}
+        {{#link-to 'admin.events.list' 'deleted' class='item'}}
+          {{t 'All Deleted'}}
+        {{/link-to}}
+        {{#link-to 'admin.events.import' class='item'}}
+          {{t 'Import'}}
+        {{/link-to}}
+      {{/tabbed-navigation}}
+    </div>
+  </div>
+</div>
 {{outlet}}
```

Comment:
Put this inside a row inside the above grid. Then wrap everything inside ` app/templates/admin/events/list.hbs` with a `div.sixteen.wide.column`